src/components/Topbar/NavigationLink.tsx

```diff
@@ -0,0 +1,23 @@
+import { Typography } from '@mui/material';
+import { NavLink } from 'react-router-dom';
+
+export const NavigationLink = (props: { path: string; label: string }) => {
+  const { path, label } = props;
+  return (
```

Comment:
nit : j'ai pris l'habitude de déstructurer dans la définition de la fonction. C'est pas super important, mais comme t'as l'air forte je suis chaud d'en discuter, surtout si t'es pas d'accord ^^

```suggestion
type NavigationLinkProps = {
  path: string;
  label: string,
}
export const NavigationLink = ({ path, label }: NavigationLinkProps) => {
  return (
```

Reply:
Mon avis est de faire le plus lisible et le plus fainéant possible :) 
Donc les deux sont des approches qui me semblent OK
Ton approche semble toutefois meilleure si on commence à avoir plus de props car le type dans le composant directement alourdit. Je pars sur ta proposition

Reply:
super, je résouds le com